analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UseTestableTimeProvider.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
```

Comment:
could you also add a test with a `DateTime` class in a `Foo.Test` namespace and using it (should be compliant)?

Reply:
Good scenarios to add. Did that.